tests/integration/span_events/test_span_events_max_samples_stored5.php

```diff
@@ -5,20 +5,20 @@
  */
 
 /*DESCRIPTION
-This is the max that can be sent with DT only.  With span limit < 7000,
+This is the max that can be sent with DT only.  With span limit > 6800,
```

Comment:
Because the traceid is now larger, the maximum number of spans that we can send to the daemon in a single message is now smaller.